internal/action/commands.go

```diff
@@ -36,11 +36,11 @@ func (s *Action) GetCommands() []*cli.Command {
 		},
 		{
 			Name:  "audit",
-			Usage: "Scan for weak passwords",
+			Usage: "Decrypt all secrets and scan for weak or shard passwords",
```

Comment:
"shard"? Did you mean "shared" as in "leaked"?

Reply:
should be "leaked".